nemo_text_processing/g2p/modules.py

```diff
@@ -254,6 +254,7 @@ def __init__(
         apply_to_oov_word=None,
         ignore_ambiguous_words=True,
         heteronyms=None,
+        use_chars=False,
```

Comment:
Could you update the IPA FastPitch config to match these new changes?

Reply:
Done